src/olympia/stats/templates/stats/reports/overview.html

```diff
@@ -6,11 +6,7 @@
     <section class="island two-up">
       <div>
         <a href="downloads/">
-        {% if waffle.flag('bigquery-download-stats') %}
```

Comment:
So, is the flag `bigquery-download-stats` on already in prod?  If it is then this doesn't seem an urgent enough change to divorce from the wider removal of the code that sets/updates weekly downloads.  If it _isn't_ enabled on prod, then is weekly downloads being processed from bigquery (even with the waffle off)?

Reply:
No, the flag isn't used/enabled/active in production.

This is a consequence of the early removal of the total downloads cron task. This template currently shows the total downloads but, because we're not updating the value anymore, the value is stale and inaccurate.

Given that we were planning on showing the weekly downloads here instead, and that weekly downloads are still updated (using the legacy infra for stats), this patch removes the condition to always show the weekly downloads.